File: [SEM6] Testowanie dotNet/GeometricFiguresFakesDataDriveTests/Geomteric/Geomteric/Circle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Geometric
{
    public class Circle : IGeometricFigure
    {
        public double R { get; set; }

        public Circle(double r)
        {
            if(r > 0)
                R = r;
            else
                throw new ArgumentOutOfRangeException("Radius less than zero");
        }

        public double countCircuit()
        {
            return 2 * Math.PI * R;
        }

        public double countSurface()
        {
            return Math.PI * R * R;
        }

        public override string ToString()
        {
            return "Circle with r = " + R;
        }
    }
}

File: [SEM6] Testowanie Java/laboratorium-1-sakovski-master/Zad4/src/Circle.java

public class Circle {
	
	public double r;
	
	public Circle(double r)	{
		this.r = r;
	}
		
	double circumfence(Circle c) {
		return 2 * Math.PI * c.r;
	}
	double poleCircle(Circle c) {
		return Math.PI * Math.pow(c.r, 2);
	}
}
